iceoryx_binding_c/test/moduletests/test_publisher.cpp

```diff
@@ -116,9 +116,9 @@ class iox_pub_test : public Test
     cpp2c_Publisher m_sut;
 };
 
```

Comment:
Please adjust copyright header!

Reply:
This was just a test if you check the headers ;-)

Reply:
You are testing me, or?! :scientist: 

Only one company per line in the copyright header.

Reply:
Arrrhhh. second